src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -1,4 +1,4 @@
-<%--
+c<%--
```

Comment:
I just noticed that this renders an extra `c` on the conversations page, probably just a typo?

Not an emergency or anything, but can you remove this whenever you get a chance? You can do it in a follow-up PR